docs/node-operators/get-started/setup-datanode.md

```diff
@@ -181,6 +181,22 @@ To update your node configuration, such as to set up ports for the APIs, edit th
 :::
 
 ### Data node configuration
+
+## Data node retention profiles
+When starting a data node, you can choose the data retention configuration for your data node, depending on the use case for the node. The retention policy details can all be fine-tuned manually, as well.
+
+There are 3 retention policy configurations: 
+* **Standard (default)**: The node retains data according to the default retention policies, which assume a data node retains some data over time, but not all data
+* **Lite**: The node retains enough data to be able to provide the latest state to clients, and produce network history segments. This mode saves enough to provide the current state of accounts, assets, balances, delegations, liquidity provisions, live orders, margin levels, markets, network limits, network parameters, node details, parties, positions 
+* **Archive**: The node retains all data
+
+To run a node that doesn't use the standard default retention, use one of the following flags when running the `init` command: 
+   
+* For a standard node, no flag
+* For an archive node, use `--archive`
+* For a lite node, use `--lite`
+
+## Generate config
```

Comment:
You might perhaps want to add that if you want to tweak the retention policy once the initial configuration has been generated, then one can set it on per-table basis in the data-node's config.toml in the sections like
```toml
  [[SQLStore.RetentionPolicies]]
    HypertableOrCaggName = "balances"
    DataRetentionPeriod = "7 days"
```

Reply:
Nice - I will add that, thank you!